.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: false
+before_install:
+  # For tests running hg command
+  # set up username
+  - echo '[ui]' > ~/.hgrc
+  - echo 'username = anybox-test' >> ~/.hgrc
+
+language: python
+
+python:
+  - "2.7"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - pip install .
+  - pip install coveralls
+  - pip install flake8
+  - pip install bzr
```

Comment:
:+1: to use requirement file as it let developers and any other tools to get the same env using the same file avoiding maintaining documentation / config files and so on.

Can you change documentation in same PR?